docs/ossm/README.md

```diff
@@ -0,0 +1,196 @@
+[Return to Project Root](../../)
+
+# Table of Contents
+
+- [User Documentation](#user-documentation)
+- [istio-csr Integration](#istio-csr-integration)
```

Comment:
Thanks for creating the docs folder! I can see this growing a lot once we start using this as a place to stage product documentation. Thinking about OSSM 2 vs 3 comparison, migration docs, etc... Would it be better to put the cert-manager/istio-csr content in its own file and have a TOC readme?